cmd/nerdctl/volume_prune_linux_test.go

```diff
@@ -17,30 +17,137 @@
 package main
 
 import (
-	"fmt"
+	"strings"
 	"testing"
 
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/icmd"
+
 	"github.com/containerd/nerdctl/v2/pkg/testutil"
 )
 
 func TestVolumePrune(t *testing.T) {
-	base := testutil.NewBase(t)
-	tID := testutil.Identifier(t)
-	base.Cmd("volume", "prune", "-a", "-f").Run()
+	// Volume pruning cannot be parallelized for Docker, since we need namespaces to do that in a way that does interact with other tests
+	if testutil.GetTarget() != testutil.Docker {
+		t.Parallel()
+	}
+
+	// FIXME: for an unknown reason, when testing ipv6, calling NewBaseWithNamespace per sub-test, in the tearDown/tearUp methods
+	// will actually panic the test (also happens with target=docker)
+	// Calling base here *first* so that it can skip NOW - does seem to workaround the problem
+	// If you have any idea how to properly do this, feel free to remove the following line and fix the underlying issue
+	testutil.NewBase(t)
+
+	subTearUp := func(tID string) {
+		base := testutil.NewBaseWithNamespace(t, tID)
+		res := base.Cmd("volume", "create").Run()
+		anonIDBusy := res.Stdout()
+		base.Cmd("volume", "create").Run()
+		base.Cmd("volume", "create", tID+"-busy").AssertOK()
+		base.Cmd("volume", "create", tID+"-free").AssertOK()
+		base.Cmd("run", "--name", tID,
+			"-v", tID+"-busy:/whatever",
+			"-v", anonIDBusy, testutil.CommonImage).AssertOK()
+	}
+
+	subTearDown := func(tID string) {
+		base := testutil.NewBaseWithNamespace(t, tID)
+		base.Cmd("rm", "-f", tID).Run()
+		//base.Cmd("volume", "rm", "-f", tID+"-busy").Run()
+		// base.Cmd("volume", "rm", "-f", tID+"-free").Run()
+		// base.Cmd("volume", "rm", "-f", anonIDFree).Run()
+		// base.Cmd("volume", "rm", "-f", anonIDBusy).Run()
```

Comment:
Is this WIP?

Reply:
No. This should have been cleaned-up - it just escaped me.

Let me do one last go through the diff and make sure there is no leftovers.

Thanks for catching it.